### ISSUE TITLE
Does not work with subdocuments

### ISSUE THREAD

**Issue by mohitleo9** [andrejewski/fakegoose#2]

**Comment by andrejewski** [comment]
Yes, subdocuments are a bit more complicated. Feel free to submit a PR, otherwise I see subdocuments like advanced query operators: only build them if there is sufficient interest.
